src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,92 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    public static final int DEFAULT_INITIAL_CAPACITY = 16;
+    public static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    public static final int CAPACITY_INCREASER = 2;
+
+    Node<K, V>[] bucketArray = new Node[DEFAULT_INITIAL_CAPACITY];
+    private int size = 0;
 
     @Override
     public void put(K key, V value) {
+        if (size + 1 > bucketArray.length * DEFAULT_LOAD_FACTOR) {
+            resize();
+        }
 
+        int bucket = bucket(key);
+        if (bucketArray[bucket] == null) { //empty bucket
+            bucketArray[bucket] = new Node<>(key, value);
+            size++;
+        } else {
+            Node<K, V> current = bucketArray[bucket];
+            for (int i = 0; i < size; i++) {
+                if (key == null ? key == current.key : key.equals(current.key)) {
+                    current.value = value; //if keys equals change old value to new
+                    return;
+                } else {
+                    if (current.next == null) { //if its end of a list put new Node
+                        current.next = new Node<>(key, value);
+                        size++;
+                        return;
+                    } else {
+                        current = current.next;
+                    }
+                }
+            }
+        }
     }
 
     @Override
     public V getValue(K key) {
+        int bucket = bucket(key);
+        Node<K, V> current = bucketArray[bucket];
```

Comment:
you use bucket variable inside this method only once, so use 
`Node<K, V> current = bucketArray[bucket(key)];`